base/database.py

```diff
@@ -275,7 +274,8 @@ class functions(object):
 
     def __new__(cls):
         '''Returns a list of all of the functions in the current database.'''
-        return builtins.list(cls.__iterate__())
+        iterable = cls.__iterate__()
```

Comment:
This doesn't need an intermediary assignment, and can just be a list-comprehension.